libapicore/httpServer.cpp

```diff
@@ -18,6 +18,12 @@ using namespace eth;
 
 httpServer http_server;
 
+// Define grayscale palette
+#define HDR0_COLOR "#E8E8E8"
```

Comment:
These can be lowercase

Reply:
> There's no need in removing the spaces in the CSS. I mean, it's not like it's that big. That being said, header0 and header1 are not good selector names in general either.

I know spaces are fine, but I'm obsessive about removing stuff machines don't care about and only server to bulk up messages. I was going to delete redundant quotes next!

What would you suggest as better selector name.